.github/workflows/linux.yml

```diff
@@ -22,6 +22,7 @@ env:
   CMAKE_VERSION: '3.29.0'
   CACHE_VERSION: 0
       
+# Execute only the latest CI run on a branch
 concurrency:
   group: ci-${{github.ref}}-linux
```

Comment:
Shouldn't we have the concurency defined right here and cancel only if branch is not master?

concurrency:
  group: mobileapp-autotests-linux # Avoid running autotests at the same time across multiple linux workflows
  cancel-in-progress: ${{ !contains(github.ref, 'master')}} # Cancel in progress for branched outside master. Always run on master